src/Microsoft.VisualStudio.Editors/SettingsDesigner/SettingsTypeCache.vb

```diff
@@ -137,6 +137,16 @@ Namespace Microsoft.VisualStudio.Editors.SettingsDesigner
             Dim qualifiedAssemblyName As String = Nothing
 
             If Not String.IsNullOrEmpty(sourceTypeName) Then
+
+                ' The configuration types represent a stable contract, for .NET scenarios pin all these type names to the desktop identities.
+                ' This will ensure that when we create required sections in app.config we'll be able to read them back.
```

Comment:
```suggestion
                ' The configuration types represent a stable contract, for .NET scenarios pin all these type names to the .NET Framework identities.
                ' This will ensure that when we create required sections in app.config we'll be able to read them back.
                ' Note that when the designer moves off of .NET Framework we may need to revisit this.
```